stardis/plasma/molecules.py

```diff
@@ -414,8 +414,8 @@ def calculate(
         linelist["level_energy_upper"] = ((linelist["e_up"].values * u.eV).cgs).value
 
         # Radiation broadening parameter is approximated as the einstein A coefficient. Vald parameters are in log scale.
-        linelist["A_ul"] = 10 ** (linelist["rad"]) / (
-            4 * np.pi
+        linelist["A_ul"] = 10 ** (
```

Comment:
There are 4 places calculating this coefficient in this PR, maybe it's not a problem but if there is a way to optimize it would be nice -- make it a properties of the linelist itself?

Reply:
I don't quite know what you're suggesting here. Certainly the molecules and the elements need to be separate functions. The solution I think outside of this PR is just to always use the shortlist calculation, since they're functionally equivalent to the shortlist, but the shortlist needs less information. 

But anyway, this is really not part of this PR. There's a bit of standardization slipped in, but this PR mostly doesn't touch the line integrated opacity calculations.